app/assets/stylesheets/arclight/modules/layout.scss

```diff
@@ -16,6 +21,32 @@
   font-size: $font-size-sm;
   height: 100%;
   padding: $spacer;
+
+  &-options {
+    display: flex;
+    align-items: center;
+    margin-top: 0.64rem;
+    margin-bottom: 0.16rem;
+    padding-top: 0.24rem;
+    padding-bottom: 0.08rem;
+  }
+
+  &-downloads {
+    padding: 0.32rem 0;
+
+    &-file {
+      padding: 0.24rem 0;
+      margin-left: -0.06rem;
+      margin-right: -0.06rem;
```

Comment:
I'm a bit curious at some of this fine-grained padding.  Is this just doing some pixel-pushing to make things look right, or has this been specified somewhere?  I wonder if there are some things built into Bootstrap that we can/should be looking towards instead (but admittedly can't offer what exactly that would be).

Reply:
I was definitely struggling with this, I can try and restructure these to use the utility classes from https://getbootstrap.com/docs/4.3/utilities/spacing/#notation